packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.11.1
+
+* Added support for image streaming to `camera_windows`.
```

Comment:
This change updates both `camera` and `camera_windows` in a single pull request. Don't we need to split this into two separate pull requests?

See: https://github.com/flutter/flutter/blob/master/docs/ecosystem/contributing/README.md#changing-federated-plugins

Reply:
pretty confusing steps, but i have split the PR

Reply:
Thanks! Agreed it's a little confusing, that's the small drawback for auto publishing packages :)